README.md

```diff
@@ -32,6 +32,15 @@ GdsApi.configure do |config|
 end
 ```
 
+By default, both `always_raise_for_not_found` and `hash_response_for_requests`
```

Comment:
The sentence about opt-in above doesn't make sense anymore. Perhaps we could rewrite this from the perspective of someone who upgrades api-adapters and sees a lot of errors in the tests?